modules/gitops-architecture-argocd-agent-intro.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * argocd-agent-architecture-overview/gitops-architecture-argocd-agent-intro.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-architecture-overview_{context}"]
+= Introduction to the Argo CD Agent architecture
+
+The Argo CD Agent introduces an opinionated mechanism for managing multiple clusters by using Argo CD in a `hub-and-spoke` configuration. When you install the Agent alongside Argo CD, it extends the familiar Argo CD web console to provide centralized visibility and control across multiple Argo CD instances.
```

Comment:
JGW-TODO: `Argo CD Agent, configured alongside Argo CD, enables the familiar Argo CD Web UI, but extends its capabilities to allow you to simultaneously configure and monitor multiple, separate Argo CD instances from a single location.`

Reply:
That's a great catch @jgwest. Thanks for clarifying it.